code/game/objects/items/implants/implant_track.dm

```diff
@@ -2,8 +2,8 @@
 	name = "tracking implant"
 	desc = "Track with this."
 	activated = FALSE
-	///for how many deciseconds after user death will the implant work?
-	var/lifespan_postmortem = 6000
+	///for how many seconds after user death will the implant work?
```

Comment:
Not true, this is still deciseconds. `SECONDS` converts it. This is an important distinction, as some code DOES use real seconds (like delta_time)

Reply:
oh. ok